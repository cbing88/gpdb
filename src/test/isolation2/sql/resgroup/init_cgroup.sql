-- start_ignore
! mkdir /sys/fs/cgroup/cpu/gpdb;
! chmod 755 /sys/fs/cgroup/cpu/gpdb;
-- end_ignore
