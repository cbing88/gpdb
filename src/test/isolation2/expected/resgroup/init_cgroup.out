-- start_ignore
! mkdir /sys/fs/cgroup/cpu/gpdb;
mkdir: cannot create directory ‘/sys/fs/cgroup/cpu/gpdb’: File exists

! chmod 755 /sys/fs/cgroup/cpu/gpdb;

-- end_ignore
